test/tests/amqp/test_amqp_racknode.py

```diff
@@ -0,0 +1,211 @@
+'''
+Copyright 2017 Dell Inc. or its subsidiaries.  All Rights Reserved.

```

Comment:
@hohene  yeah, my misunderstanding of 'or its subsidiaries' 
@nortonluo missing "(c)" in this line